src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -29,7 +37,7 @@
   }
 
   &.#{$prefix}-controlbar-visible {
-    bottom: 5em;
-    transition: bottom $animation-duration-short ease-in;
+    height: 70%;
```

Comment:
Not sure I understand the question.

Are you saying that in some case the control bar and the overlay could superpose each other? If that's the issue, in my testing I found that those case were rare, (really small player) and the default was not really better as it tended to superpose with the description.

I wished I was able to not use a fixed height and just have a `bottom` and `top` property but that didn't work for me.